src/components/TitleItem.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <h1>
+    {{ props.title }}
+  </h1>
+</template>
+
+<script setup lang="ts">
+  const props = defineProps(["title"])
```

Comment:
We are using TypeScript. So please define props types using interface. 
Here is an example:
```ts
<script setup lang="ts">
interface Props {
  foo: string
  bar?: number
}

const props = defineProps<Props>()
</script>
```
If you are not comfortable with TS or have any questions feel free to ping me in chat or in private and we can figure it out